WORKSPACE

```diff
@@ -131,8 +131,12 @@ maven_install(
     ],
 )
 
+android_sdk_repository(
```

Comment:
Wait, why do you need this in the root workspace?

Reply:
When I tried to run the tests locally it complained about an android sdk not having been configured. Adding this solved the problem. I can't recall the exact error message.

Reply:
Seems that this was added at head so it's a moot point now.